certif/tests/acceptance/team/list/invitations_test.js

```diff
@@ -81,6 +81,27 @@ module('Acceptance | Team | Invitations', function (hooks) {
           .dom(screen.getByText(this.intl.t('pages.team-invitations.notifications.success.invitation-cancelled')))
           .exists();
       });
+
+      module('when an error occurs', function () {
+        test('displays an error notification', async function (assert) {
+          // given
+          this.server.create('certification-center-invitation', {
+            id: 15,
+            certificationCenterId: 1,
+            email: 'anna.liz@example.net',
+            updatedAt: new Date('2023-11-30'),
+          });
+
+          const screen = await visit('/equipe/invitations');
```

Comment:
On a ici un contexte tres specifique, l'id 15 est choisi "au hasard". 
On pourrait expliciter dans le test l'erreur:
 ```suggestion 
        this.server.delete(`/certification-center-invitations/15`, (schema, request) => new Response(500);
         const screen = await visit('/equipe/invitations');
```

Reply:
Je suis d'accord, ça enléverait ces lignes côté [config.js](https://github.com/1024pix/pix/pull/7554/commits/0a42081aa39f0d9be61cba3afb4cfbcaa21ed480#diff-e8c699b64b115d99fe24c1b391e246a2cb7a7afee4df349be665b5809de98348R396-R398)